### PR TITLE
docs: update structure

### PR DESCRIPTION
**Pull request by woile**
related #388

### CODE REVIEW

File: site/src/system.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-# `system`
+# Cross-platform: `system`
```
Comment (roberth) [inline site/src/system.md:2]: I'd like to reserve "cross" for cross-compilation if possible. What do you think of

```suggestion
# Multi-platform: `system`
```

File: site/src/SUMMARY.md [hunk 2]
```diff
@@ -1,17 +1,18 @@
 # Summary
 
 - [Introduction](./README.md)
+- [Getting Started](./getting-started.md)
+- [Cheat Sheet](./cheat-sheet.md)
+- [Tutorials]()
```
Comment (roberth) [inline site/src/SUMMARY.md:6]: These aren't *really* tutorials yet, but that's perhaps a separate issue.

File: site/src/cheat-sheet.md
Comment (roberth) [inline site/src/cheat-sheet.md:1]: Should add a netlify redirects file

The site build is declared in https://github.com/hercules-ci/flake.parts-website/blob/main/site/site-module.nix

**Comment by roberth** [comment]
bors try

File: site/src/cheat-sheet.md
Comment (woile) [reply site/src/cheat-sheet.md:1]: I'm not sure how to achieve that. Should I add a `_redirects` file and then after this line

https://github.com/hercules-ci/flake.parts-website/blob/fab7c94ce6e57990b1d7dd9cb8b55a9cb25ba477/site/site-module.nix#L62

add a 

```
cp _redirects $out
```

would that work?

**Comment by roberth** [comment]
bors try

Let's test the redirects

File: site/src/SUMMARY.md [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
+  - [Best Practices for Module Writing](./best-practices-for-module-wrciting.md)
```
Comment (roberth) [inline site/src/SUMMARY.md:7]:
```suggestion
  - [Best Practices for Module Writing](./best-practices-for-module-writing.md)
```

**Comment by roberth** [comment]
bors try

Redirect worked. Now testing the typo.

**Comment by roberth** [comment]
- #216 is slowing us down :(. Do you happen to know a better link checker that supports fragments?

**Review by roberth — APPROVED** [review]

**Comment by roberth** [comment]
All good now. Thanks!

bors r+

**Comment by woile** [comment]
> Do you happen to know a better link checker that supports fragments?

I would keep an eye on this issue: https://github.com/lycheeverse/lychee/issues/185

**Pull request merged**